11ty/_includes/layouts/base.njk

```diff
@@ -1,17 +1,20 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link rel="preconnect" href="https://use.typekit.net/" crossorigin />
+    <link rel="dns-prefetch" href="https://use.typekit.net/" />
+    <link rel="preconnect" href="https://p.typekit.net/" crossorigin />
+    <link rel="dns-prefetch" href="https://p.typekit.net/" />
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-
     {# Use title with path, or append a space to the page title to avoid collpasing with the meta title #}
     {% set pageTitle = titleWithPath or title + ' ' or '' %}
     <title>{{ pageTitle + metadata.title }}</title>
     <meta
       name="description"
       content="{{ renderData.description or description or metadata.description }}"
     >
-    <link rel="stylesheet" href="https://use.typekit.net/qlo3dpu.css" rel="preload"/>
```

Comment:
The `preload` hint here is removed, as this is not the preferred value for `preload`. In this case, that would be an entirely separate `<link>` tag like this:

```html
<link rel="preload" as="style" href="https://use.typekit.net/qlo3dpu.css" crossorigin />
```

Even so, this hint would be so close to the stylesheet in the <head> that it would have no meaningful effect.